kubernetes/loculus/templates/loculus-preprocessing-deployment.yaml

```diff
@@ -3,10 +3,7 @@
     "http://host.k3d.internal:8079"
     "http://loculus-backend-service:8079"
 }}
-{{- $organismcloakHost := $.Values.environment | eq "server" | ternary
-    (printf "https://authentication%s%s" $.Values.subdomainSeparator $.Values.host)
-    "http://loculus-keycloak-service:8083"
-}}
+{{- $keycloakHost := $testconfig | ternary "http://localhost:8083" "http://loculus-keycloak-service:8083" }}
```

Comment:
Thanks for the PR - this is giving an error in tests atm - testConfig not defined